src/scripts.js

```diff
@@ -311,22 +318,25 @@ function createMissingIngredientsModal(recipeID) {
 function closePopUp(event) {
   if (event.target.id === 'dismissButton') {
     popupSuccess.style.display = 'none';
+    popupError.style.display = 'none';
   }
 }
 
-function addRecipesToPantry(recipeID) {
+function addToPantry() {
+  const recipeId = Number(addIngredientsBtn.dataset.recipeId);
   toggleIngredientsBtn();
-  addAllIngredients(recipeID, store.user);
+  addAllIngredients(recipeId, store.user);
 
   setTimeout(() => {
     addIngredientsBtn.classList.remove('btn-loading');
     missingIngredientModal.style.display = 'none';
-    addIngredientSuccessPopup.style.display = 'block';
   }, 2000);
 }
 
-function displayPopUp() {
+function removeFromPantry() {
+  const recipeId = Number(addIngredientsBtn.dataset.recipeId);
   addIngredientSuccessPopup.style.display = 'none';
+  removeIngredientsFromPantry(recipeId, store.user);
```

Comment:
Remove from pantry now gets invoked when a user clicks on the `cook recipe btn` on the modal. It also uses the recipe id of the add ingredients btn the same way as the function above.